.gitignore

```diff
@@ -150,3 +150,5 @@ certs
 
 # nx
 .nx
+
+.codegpt
```

Comment:
perhaps this should be set as local gitignore instead of a repo one?

Reply:
does it matter?